.github/workflows/tests.yaml

```diff
@@ -1,25 +1,18 @@
-on: [push, pull_request]
+on: [push]
 
 jobs:
   tests:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout repository and submodules
-      uses: actions/checkout@v2
-      with:
-        submodules: recursive
+      - uses: actions/checkout@v3
 
-    - name: Install nix 2.3.6
-      uses: cachix/install-nix-action@v13
-      with:
-        install_url: https://releases.nixos.org/nix/nix-2.3.6/install
-        nix_path: nixpkgs=channel:nixos-unstable
+      - name: Define Foundry
```

Comment:
```suggestion
      - name: Install Foundry
```

Reply:
Is it actually installing anything in this block or is it defining where to get foundry/forge?

Reply:
Excepting I'm not understanding how it works I feel the naming Chris changed is more appropriate.
Actually your suggestion is getting back to what it was.

Reply:
I've checked it against here: https://github.com/marketplace/actions/foundry-toolchain

Reply:
I think the github action is installing foundry

Reply:
Hmm if that is the case, then is right, why are we also adding `forge install`?

Reply:
Ahh ok got it. Then yes, these suggested names (similar to the original ones) make more sense.
Sorry about the confusion.

Reply:
no worries! got confused as well initially.